hashtable/hashtable.py

```diff
@@ -65,14 +100,24 @@ def get(self, key):
 
         Implement this.
         """
+        index =  self.hash_index(key)
+        cur = self.storage[index]
+
+        while cur != None:
+            if cur.key == key:
+                return cur.value
+            cur = cur.next
+
+        return None
 
     def resize(self):
         """
         Doubles the capacity of the hash table and
         rehash all key/value pairs.
-
         Implement this.
         """
+        self.storage = self.storage + [None] * self.capacity
```

Comment:
Do you mean to erase all of your prior storage upon resizing?

Reply:
no, not really. I only mean to add up the size. is that what the code is doing?

Reply:
ohhh that code was written before the new upstream changes, that's why I wrote it like that.

Reply:
That makes sense! This wasn't part of yesterdays MVP, but just something I noticed 👍